### ISSUE TITLE
[EXE] Implementing U-Net for Image Segmentation

### ISSUE THREAD

**Issue by msethi006** [gimseng/99-ML-Learning-Projects#165]
**Learning Goals**
1.) Downloading dataset from PlantVillage and doing annotations for segmentation.
2.) Implementing the U-Net architecture from scratch in Tensorflow.

**Exercise Statement**
Implement U-Net architecture for Plant Leaf Disease Detection.

**Prerequisites**
Deep-Learning, Convolutional Neural Networks.

**Data source/summary**
PlantVillage Dataset (https://www.kaggle.com/emmarex/plantdisease)

**Suggest/Propose Solutions**
I have implemented the solution and would like to contribute here. If someone could assign me the issue, I can create a pull request then.

**Comment by Namangarg110** [comment]
I have worked a similar project on electrical substation masking. And I would like to contribute with the same
